tests/security_features_test.go

```diff
@@ -280,6 +280,7 @@ var _ = Describe("[Serial][sig-compute]SecurityFeatures", Serial, decorators.Sig
 	})
 	Context("Disabling the custom SELinux policy", func() {
 		var policyRemoved = false
+		var featureGateEnabled = false
```

Comment:
Could we evaluate this in a BeforeEach statement with `checks.HasFeature`

Reply:
This is just a poorly named variable, I'll rename to something like `featureGateEnabledByTest`.
Its purpose is to tell `AfterEach` that some cleanup is needed. Same for the other variable above.